### DISCUSSION TITLE
How to reindex skipped files after repair

### DISCUSSION THREAD

**Discussion by anthraxn8b** [photoprism/photoprism#4637] [Q&A]
I would like to index all files that were skipped.
This can be because of the right file ending but the wrong type but also vice versa and so on.
Is there a simple solution for this?
I am running on a PI4B and don not want to reindex the complete 250.000 files...

**Comment by lastzero** [comment]
If you have fixed (modified) them, this will be detected automatically when your library is indexed, so you do not need to force a complete rescan.
